lib/rules/jsx-sort-props.js

```diff
@@ -107,7 +107,7 @@ const generateFixerFunction = (node, context, reservedList) => {
       });
     });
 
-    fixers.sort((a, b) => a.range[0] < b.range[0]);
+    fixers.sort((a, b) => Math.sign(b.range[0] - a.range[0]));
```

Comment:
I’m not sure the Math.sign is needed at all; comparators can return any negative or positive number.